packages/commcare/test/index.js

```diff
@@ -319,6 +326,107 @@ describe('getCases', () => {
       },
     ]);
   });
+
+  it('should fetch cases with pagination', async () => {
```

Comment:
This is in the `getCases` describe block, but it's not a `getCases ` test.

This is a test of `get`. 

Also, the test name is wrong. We don't care about `fetching cases` here, we care about `GET with pagination`.

Please restructure the tests so that the right functions are tested in the right place. And for any `get` tests, make sure the names are appropriate.

Reply:
I think we need many more tests on pagination too:

```
should not add limit and offset to the parameters of the first request if the user does not pass them
should fetch the next page if the response has a meta.next property
should not auto-fetch if the user sets an offset
should respect the user's limit while paginating
should call the callback once for a single request
should call the callback once per page
should stop paginating after the first error
should not force a non-paginated response into an array
```

Can you think of any more?